tests/Fixtures/SucceedingStringLengthValidator.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+declare( strict_types = 1 );
+
+namespace WMDE\Fundraising\Frontend\Tests\Fixtures;
+
+use WMDE\Fundraising\Frontend\Validation\StringLengthValidator;
+use WMDE\Fundraising\Frontend\Validation\ValidationResult;
+
+/**
+ * @license GNU GPL v2+
+ * @author Gabriel Birke < gabriel.birke@wikimedia.de >
+ */
+class SucceedingStringLengthValidator extends StringLengthValidator {
```

Comment:
This is never used. I assume, this slipped into the commit while you were already working on another task?